src/adaptors/http1/http1_client.c

```diff
@@ -1218,6 +1406,26 @@ uint64_t qdr_http1_client_core_link_deliver(qdr_http1_adaptor_t    *adaptor,
     _client_response_msg_t *rmsg = DEQ_TAIL(hreq->responses);
     assert(rmsg && rmsg->dlv == delivery);
 
+    if (hconn->cfg.aggregation != QD_AGGREGATION_NONE) {
```

Comment:
Pls add comment explaining that responses are saved and not encoded until after the request has been accepted.

Reply:
Will do